backend/wmg/api/wmg-api-v2.yml

```diff
@@ -0,0 +1,821 @@
+openapi: 3.0.3
+info:
+  version: "1.0.0"
+  title: Chan Zuckerberg Initiative cellxgene Where's My Gene (WMG) API
+  description: >-
+    This API is for internal use only by WMG web client.
+servers:
+  - description: Local
+    url: /
+  - description: Production environment
+    url: https://api.cellxgene.cziscience.com/
+  - description: Development environment
+    url: https://api.dev.single-cell.czi.technology/
+  - description: Staging environment
+    url: https://api.staging.single-cell.czi.technology/
+
+paths:
+  /primary_filter_dimensions:
```

Comment:
@atarashansky - I originally encoded it as follows (see this [commit](https://github.com/chanzuckerberg/single-cell-data-portal/pull/4887/commits/f5c018feba840cf0fd9ab4fedb65e5347fe7154a)):

```
# V1 API
add_api(base_path="/wmg", spec_file="wmg/api/wmg-api.yml")

# V2 API
add_api(base_path="/wmg", spec_file="wmg/api/wmg-api-v2.yml")
```

`wmg-api-v2.yml` had paths `/v2/query` instead of `/query` as it does in this latest commit. However, I was getting exceptions in test because `/wmg` was mapped to 2 spec files and that is not allowed.

I think the url routing logic requires all `base_path` _values to be unique_. 

So rather than have a dummy `base_path` like `new_wmg` or something like that, I pulled the version number into the `base_path` to make it unique. The version of the spec file then is encoded in the spec file name.

This allows creation of different spec files for different versions of the API as well (rather than have a single spec file contain versioned url paths like `/v1/query` and `/v2/query` etc - which might become difficult to read)

Reply:
I didn't know about the uniqueness requirement. This makes perfect sense. Thanks for explaining!